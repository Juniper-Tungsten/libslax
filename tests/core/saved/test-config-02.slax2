version 1.1;

match configuration {
    apply-templates system/host-name;
}

match host-name {
    <hello> .;
}
