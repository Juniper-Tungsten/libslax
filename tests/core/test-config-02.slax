version 1.0;

match configuration {
    apply-templates system/host-name;
}

match host-name {
    <hello> .;
}
