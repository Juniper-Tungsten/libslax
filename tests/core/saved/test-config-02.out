<?xml version="1.0"?>
<hello>puff-the-magic-dragon</hello>
